skyve-war/src/main/java/modules/admin/Jobs/Jobs.xml

```diff
@@ -3,6 +3,11 @@
 			xmlns="http://www.skyve.org/xml/document"
 			xsi:schemaLocation="http://www.skyve.org/xml/document ../../../schemas/document.xsd"
 			xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<documentation>
+		<![CDATA[
+		Live monitoring dashboard for currently executing jobs.
```

Comment:
Non-persistent document that provides a view for live monitoring of currently executing jobs, viewing the history of completed jobs, and maintaining the schedule for future jobs.